abel/onion_bordas.py

```diff
@@ -0,0 +1,158 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import numpy as np
+from scipy.ndimage.interpolation import shift
+
+################################################################################
+#
+# Onion peeling algorithm, also known as "back projection"
+#
+# This algorithm was adapted by Dan Hickstein from the original Matlab 
+# implementation, created by Chris Rallis and Eric Wells of 
+# Augustana University, and described in this paper:
+#
+#  http://scitation.aip.org/content/aip/journal/rsi/85/11/10.1063/1.4899267
```

Comment:
can you also put these references into the main onion_peeling_transform docstring?